webpack/assets/javascripts/react_app/components/BreadcrumbBar/__tests__/BreadcrumbBarReducer.test.js

```diff
@@ -7,6 +7,7 @@ import {
 } from '../BreadcrumbBarConstants';
 import reducer from '../BreadcrumbBarReducer';
 
+import { testReducerSnapshotWithFixtures } from '../../../common/testHelpers';
```

Comment:
how about use it in other reducers tests?

Reply:
Hi  @amirfefer 
I moved `PasswordStrength` reducer tests too.
I am not sure I can move `factCharts`, `notifications` and others as they are passing a previous state in their tests while the `testReducerSnapshotWithFixtures` gets only `action`.